src/appleseed.bench/mainwindow/rendering/renderwidget.cpp

```diff
@@ -205,7 +207,16 @@ void RenderWidget::highlight_tile(
 
     // Draw a bracket around the tile.
     const size_t BracketExtent = 5;
-    const std::uint8_t BracketColor[3] = { 255, 255, 255 };
+    std::uint8_t BracketColor[3] = { 255, 255, 255 }; // white
+    if (thread_index >= 0) {
+        // Choose one color per thread (see https://www.shadertoy.com/view/wlKXDm).
+        float t = TwoPi<float>() * (thread_index / static_cast<float>(nb_threads));
```

Comment:
We don't have much control over colors here. Have you considered creating a color palette and picking colors in it like this : 
`color_index = thread_index % nb_threads % color_palette_size` ? 
Like this we don't cast anything and we can choose the colors :)

Reply:
Moreover, you can simply drop the `nb_threads` value if you use a color palette.

Reply:
As discussed in #dev, I think this is a reasonable solution for the time being.